src/bus.rs

```diff
@@ -130,6 +130,43 @@ pub trait UsbBus: Sync + Sized {
     }
 }
 
+// TODO: merge this into UsbBus with the next breaking change
```

Comment:
I think it's perfectly fine to just add everything to `UsbBus`, it's versioned, and that's what 0.x versions are for.